keyboards/handwired/6key/6key.h

```diff
@@ -0,0 +1,15 @@
+#ifndef SIXKEY_H
+#define SIXKEY_H
+
+#include "quantum.h"
+
+#define LAYOUT( \
+      k00, k01, k02,    \
+      k03, k04, k05     \
+) \
+{ \
+    { k00, k01, k02 },  \
+    { k03, k04, k05 }   \
+} 
+
+#endif
```

Comment:
```suggestion
/* Copyright %YEAR% %YOUR_NAME%
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 2 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

#pragma once

#include "quantum.h"

#define LAYOUT( \
      k00, k01, k02,    \
      k03, k04, k05     \
) \
{ \
    { k00, k01, k02 },  \
    { k03, k04, k05 }   \
} 

```